etc/Env.sh

```diff
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR/..
```

Comment:
```suggestion
# Make sure we are on the correct folder before beginning
cd "$(dirname $(readlink -f $0))"
```

Since this script (`Env.sh`) is intended to be run and not sourced, I am partial to what we have in the `build_openroad.sh` from ORFS. Also, `$BASH_SOURCE` is bash-specific; macOS comes with `zsh` by default for example.